storage/storage.h

```diff
@@ -24,6 +24,9 @@
 #include <stdint.h>
 #include "secbool.h"
 
+// The length of the external salt in bytes.
+#define EXTERNAL_SALT_SIZE 32
```

Comment:
Minimum block size on the SD card is 512 bytes. Surely it makes no sense to use 4096 bits of entropy, but we might consider increasing this to 64 bytes?

Reply:
Well, I think that when the salt enters PBKDF2-HMAC-SHA256, the entropy eventually gets hashed down to 256 bits anyway. So using a higher value here would be giving a false sense of greater security.

Reply:
OK